labs/SizerBase/samples/SizerBase.Samples/GridSplitter.md

```diff
@@ -0,0 +1,22 @@
+---
+title: GridSplitter
+author: mhawker
+description: The GridSplitter control provides an easy-to-use Splitter that redistributes space between columns or rows of a Grid Control.
+keywords: ContentSizer, SizerBase, Control, Layout, Expander
+dev_langs:
+  - csharp
+category: Controls
+subcategory: Layout
+---
+
+# GridSplitter
+
+<!-- TODO: Link docs -->
+The control automatically detects the targeted columns/rows to resize, while dragging the control it starts to resize the columns/rows and redistributes space between columns/rows,
+you can manually specify the Resize Direction Auto, Column, Row and the Resize Behavior to select which columns/rows to resize.
+# Examples
+
+GridSplitter control will resize the targeted columns or rows.
+
+
+> [!SAMPLE GridSplitterPage]
```

Comment:
```suggestion

# Examples

GridSplitter control will resize the targeted columns or rows.

> [!SAMPLE GridSplitterPage]

```

Clean-up lines, we should have blanks around headers and lists for better markdown parsing.

Reply:
Yes, Cleaned up the lines and unnecessary links